### PR TITLE
Enhancement log-recerive e2e test

### PR DESCRIPTION
**Pull request by arugal**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: docs/en/setup/backend/configuration-vocabulary.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ core|default|role|Option values, `Mixed/Receiver/Aggregator`. **Receiver** mode
 | - | - | instanceNameMaxLength| Max length limitation of service instance name. The max length of service + instance names should be less than 200.|SW_INSTANCE_NAME_MAX_LENGTH|70|
 | - | - | endpointNameMaxLength| Max length limitation of endpoint name. The max length of service + endpoint names should be less than 240.|SW_ENDPOINT_NAME_MAX_LENGTH|150|
 | - | - | searchableTracesTags | Define the set of span tag keys, which should be searchable through the GraphQL. Multiple values should be separated through the comma. | SW_SEARCHABLE_TAG_KEYS | http.method,status_code,db.type,db.instance,mq.queue,mq.topic,mq.broker|
-| - | - | searchableLogsTags | Define the set of log tag keys, which should be searchable through the GraphQL. Multiple values should be separated through the comma. | SW_SEARCHABLE_LOGS_TAG_KEYS | - |
+| - | - | searchableLogsTags | Define the set of log tag keys, which should be searchable through the GraphQL. Multiple values should be separated through the comma. | SW_SEARCHABLE_LOGS_TAG_KEYS | level,logger,thread |
```
Comment (wu-sheng) [inline docs/en/setup/backend/configuration-vocabulary.md:33]: What do these mean? I don't remember we have LAL(Log Analysis Language) to separate keys from the log text.

Comment (arugal) [reply docs/en/setup/backend/configuration-vocabulary.md:33]: [logback](https://github.com/apache/skywalking/blob/e138d29778fb458a6fad5f3fd82121c7bba3cc95/apm-sniffer/apm-toolkit-activation/apm-toolkit-logback-1.x-activation/src/main/java/org/apache/skywalking/apm/toolkit/activation/log/logback/v1/x/log/GRPCLogAppenderInterceptor.java#L83-L88) and [log4j](https://github.com/apache/skywalking/blob/e138d29778fb458a6fad5f3fd82121c7bba3cc95/apm-sniffer/apm-toolkit-activation/apm-toolkit-log4j-2.x-activation/src/main/java/org/apache/skywalking/apm/toolkit/activation/log/log4j/v2/x/log/GRPCLogAppenderInterceptor.java#L82-L87) will report these tags.

Comment (wu-sheng) [reply docs/en/setup/backend/configuration-vocabulary.md:33]: I see. Make sense.

File: test/e2e/e2e-test/docker/kafka/docker-compose.log.yml [hunk 2]
```diff
@@ -31,33 +31,34 @@ services:
...
+
+  provider:
+    extends:
+      file: ../base-compose.yml
+      service: provider
+    environment:
+      SW_GRPC_LOG_SERVER_HOST: sw-satellite
+    depends_on:
+      oap:
+        condition: service_healthy
```
Comment (kezhenxu94) [inline test/e2e/e2e-test/docker/kafka/docker-compose.log.yml:62]: Better also add `sw-satellite` here

File: test/e2e/e2e-test/docker/kafka/docker-compose.log.yml [hunk 3]
```diff
@@ -31,33 +31,34 @@ services:
...
-      JAVA_OPTS: >-
-        -javaagent:/jacoco/jacocoagent.jar=classdumpdir=/jacoco/classes/provider,destfile=/jacoco/provider.exec,includes=org.apache.skywalking.*,excludes=org.apache.skywalking.apm.dependencies.*
-         -Dbootstrap.service=broker-a:9092,broker-b:9092
-    healthcheck:
-      test: ["CMD", "sh", "-c", "nc -nz 127.0.0.1 8080"]
-      interval: 5s
-      timeout: 60s
-      retries: 120
+      - ./satellite_config.yaml:/configs/satellite_config.yaml
+    command: ["start"]
```
Comment (kezhenxu94) [inline test/e2e/e2e-test/docker/kafka/docker-compose.log.yml:45]: Forget to modify this? Should be `/sw-satellite` or remove this line?

Comment (arugal) [reply test/e2e/e2e-test/docker/kafka/docker-compose.log.yml:45]: image

The full command is `/sw-satellite start`, `/sw-satellite` in `ENTRYPOINT`.

File: test/e2e/e2e-test/docker/kafka/docker-compose.log.yml (hunk 2 above)
Comment (arugal) [reply test/e2e/e2e-test/docker/kafka/docker-compose.log.yml:67]: Thx Zhenxu :)

**Review by wu-sheng — APPROVED** [review]
@kezhenxu94 located 2 issues for the current implementation. @arugal @EvanLjp @Fine0830 
1. Error status in query condition and result list are not necessary. Log level exists in the tag only. @Fine0830 needs to remove them, and add tags as a column in the list page.
2. Exception stacks are not collected from current log plugins.

@kezhenxu94 will make these adjustment. These should make 8.4.0 release better.

**Pull request merged**